flink-runtime/src/main/java/org/apache/flink/runtime/io/network/NettyShuffleServiceFactory.java

```diff
@@ -111,6 +118,14 @@ static NettyShuffleEnvironment createNettyShuffleEnvironment(
 
         FileChannelManager fileChannelManager =
                 new FileChannelManagerImpl(config.getTempDirs(), DIR_NAME_PREFIX);
+        if (LOG.isInfoEnabled()) {
+            LOG.info(
+                    "Created a new {} for storing result partitions of BLOCKING shuffle. Used directories: {}.",
+                    FileChannelManager.class.getSimpleName(),
+                    Arrays.stream(fileChannelManager.getPaths())
+                            .map(File::getAbsolutePath)
+                            .collect(Collectors.joining(", ")));
```

Comment:
same as above

Reply:
answered above, I don't think this is a good idea from operations point of view